packages/manager/tools/webpack-dev-server/src/config.js

```diff
@@ -21,7 +21,11 @@ module.exports = (env) => {
 
   const sso = new Sso(region, envConfig);
 
-  if (yn(env.local2API) || yn(process.env.npm_package_config_local2API)) {
+  if (
```

Comment:
Question: Why are we adding npm_config_local2API here ?

Reply:
see my last comment. In this way, I can activate local2API in the user .npmrc file

Reply:
I removed the file form this commit @sachinrameshn